lib/tutorials/en_US/cookies.md

```diff
@@ -46,10 +46,19 @@ reply('Hello').state('data', { firstVisit: false });
 
 In this example, hapi will reply with the string `Hello` as well as set a cookie named `data` to a base64 encoded string representation of the given object.
 
+## Clearing a cookie
+The cookie can be cleared by calling the `unstate()` method on the [`response`](/api#response-object) object:
+
+```javascript
+reply('Hello').unstate('data', { firstVisit: false });
```

Comment:
`firstVisit` is just an example of some previous state. I used this property name to match what was already there on the Setting a Cookie section of the tutorial.